src/webui/www/private/scripts/mocha-init.js

```diff
@@ -38,6 +38,41 @@
    ----------------------------------------------------------------- */
 "use strict";
 
+window.qBittorrent ??= {};
+window.qBittorrent.Dialog ??= (() => {
+    const exports = () => {
+        return {
+            baseModalOptions: baseModalOptions
+        };
+    };
+
+    const baseModalOptions = Object.assign(Object.create(null), {
+        addClass: "modalDialog",
+        collapsible: false,
+        cornerRadius: 5,
+        draggable: true,
+        footerHeight: 20,
+        icon: "images/qbittorrent-tray.svg",
+        loadMethod: "xhr",
+        maximizable: false,
+        method: "post",
+        minimizable: false,
+        padding: {
+            top: 15,
+            right: 10,
+            bottom: 15,
+            left: 5
+        },
+        resizable: true,
+        width: 480
+    });
+
+    window.qBittorrent.Cache.deepFreeze(baseModalOptions);
```

Comment:
It seems `deepFreeze` isn't related to `qBittorrent.Cache` so it shouldn't be exposed/exported.
Ideally there could be another file (object.js) and namespace called `qBittorrent.Object` that expose `deepFreeze`. However I'm concerned that would create more dependency between them and might cause circular dependency later.

Maybe just call `Object.freeze()` on both `baseModalOptions` & `padding`, or duplicate `deepFreeze` as an internal function within `qBittorrent.Dialog`?

Reply:
Alright, I decided to just duplicate it for now.